base/lttagging.dtx

```diff
@@ -371,6 +371,17 @@
 \NewSocket{tagsupport/toc/leaders/after}{0}
 %    \end{macrocode}
 % \end{socketdecl}
+% 
+% \subsubsection{Tagging support for marginpar}
+% 
+% \begin{socketdecl}{tagsupport/marginpar/begin,tagsupport/marginpar/end}
+% \changes{v1.0n}{2025/02/21}{move marginpar sockets}
```

Comment:
Nothing important. The date of this dtx, `lttagging.dtx` was not updated together.
https://github.com/latex3/latex2e/blob/dfa851006eb94779f7266f5670911ed59b4d83cc/base/lttagging.dtx#L35-L36